lightning/src/ln/channel.rs

```diff
@@ -5393,13 +5422,15 @@ impl<'a, Signer: Sign, K: Deref> ReadableArgs<&'a K> for Channel<Signer>
 
 		let mut announcement_sigs = None;
 		let mut target_closing_feerate_sats_per_kw = None;
+		let mut channel_creation_height = None;
 		read_tlv_fields!(reader, {
 			(0, announcement_sigs, option),
```

Comment:
unrelated to this diff, but where does `option` get set?

Reply:
its actually a macro argument, see `lightning/src/util/ser_macros.rs` in macros like `check_tlv_order` or `encode_tlv`.